### PR TITLE
chore: Update user-guide documentations

### PR DESCRIPTION
**Pull request by Rolika4**
* Update environments block

## Description

Please include a summary of the change and why it is needed.

Fixes #(issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Pull Request contain one commit. I squash my commits.

### CODE REVIEW

File: docs/user-guide/add-cd-pipeline.md [hunk 1]
```diff
@@ -43,76 +42,74 @@ the lower-case letters, numbers and inner dashes.
...
 
-### The Applications Menu
+    2. Specify the application parameters:
 
-The **Pipeline** tab of the **Create CD Pipeline** menu is presented below:
+        * **Branch** - Select the application branch from the drop-down menu.
 
-    ![Environment applications](../assets/user-guide/edp-portal-cd-pipeline-applications.png "Environment applications")
+        * **Promote in pipeline** - Select the this check box in order to transfer the application from one to another stage
+        by the specified codebase Docker branch. If the **Promote in pipeline** check box is not selected,
```
Comment (SergK) [inline docs/user-guide/add-cd-pipeline.md:56]: please replace Docker branch just with brach

File: docs/user-guide/add-cd-pipeline.md [hunk 2]
```diff
@@ -43,76 +42,74 @@ the lower-case letters, numbers and inner dashes.
...
+        the same codebase Docker stream will be deployed regardless of the stage, i.e. the codebase Docker stream input,
```
Comment (SergK) [inline docs/user-guide/add-cd-pipeline.md:57]: remove Docker as well

File: docs/user-guide/add-cd-pipeline.md [hunk 3]
```diff
@@ -43,76 +42,74 @@ the lower-case letters, numbers and inner dashes.
...
+        which was selected for the pipeline, will always be used.
 
-1. Select the necessary application from the **Mapping field name** drop-down menu and click **Add**.
-2. Specify the application parameters:
+        :::note
+          If there is another deployed environment stage with the respective codebase Docker stream (equal image stream as an OpenShift term), the pattern combination will be as follows: `[pipeline name]-[stage name]-[application name]-[verified]`.
```
Comment (SergK) [inline docs/user-guide/add-cd-pipeline.md:61]: do not use Docker

File: docs/user-guide/add-cd-pipeline.md [hunk 4]
```diff
@@ -43,76 +42,74 @@ the lower-case letters, numbers and inner dashes.
...
 
-        * **Cluster** - Choose the cluster to deploy the stage in;
-        * **Stage name** - Enter the stage name;
-        * **Namespace** - Specify the Kubernetes namespace where the resources will be deployed in. By default, this field is pre-populated automatically but keep in mind that the namespace name must be no longer than 63 symbols;
-        * **Description** - Enter the description for this stage;
-        * **Trigger type** - Select the trigger type. The key benefit of the automatic deploy feature is to keep environments up-to-date. The available trigger types are _Manual_ and _Auto_. When the _Auto_ trigger type is chosen, the environment will initiate automatically once the image is built. _Manual_ implies that user has to perform deploy manually by clicking the **Deploy** button in the environment menu. Please refer to the [Architecture Scheme of CD Pipeline Operator](https://github.com/epam/edp-cd-pipeline-operator/blob/master/docs/arch.md) page for additional details.
+        :::note
```
Comment (SergK) [inline docs/user-guide/add-cd-pipeline.md:88]: hm need to check afaik this is already addressed

File: versioned_sidebars/version-3.9.0-sidebars.json [hunk 5]
```diff
@@ -383,8 +383,7 @@
           "collapsed": false,
           "items": [
             "user-guide/add-cd-pipeline",
-            "user-guide/manage-environments",
-            "user-guide/add-quality-gate"
```
Comment (SergK) [inline versioned_sidebars/version-3.9.0-sidebars.json:387]: why did you remove this?

**Review by SergK — CHANGES_REQUESTED** [review]

File: versioned_sidebars/version-3.9.0-sidebars.json (hunk 5 above)
Comment (Rolika4) [reply versioned_sidebars/version-3.9.0-sidebars.json:387]: remove just "user-guide/add-quality-gate" and "," in 386 string
